### PR TITLE
skel: misc: Replace args and config methods

### PR DESCRIPTION
**Pull request by jstar1**
Removes all instances of args and config.

fixes: #164

### CODE REVIEW

File: dffml/skel/source/REPLACE_IMPORT_PACKAGE_NAME/misc.py [hunk 1]
```diff
@@ -4,10 +4,11 @@
 from dffml.repo import Repo
 from dffml.source.source import BaseSourceContext, BaseSource
 from dffml.util.cli.arg import Arg
-from dffml.util.entrypoint import entrypoint
+from dffml.util.entrypoint import entry_point
```
Comment (pdxjohnny) [inline dffml/skel/source/REPLACE_IMPORT_PACKAGE_NAME/misc.py]: `entry_point` was recently renamed to `entrypoint` throughout the rest of the codebase.

```suggestion
from dffml.util.entrypoint import entrypoint
```

File: dffml/skel/model/REPLACE_IMPORT_PACKAGE_NAME/misc.py [hunk 2]
```diff
@@ -11,10 +11,11 @@
 from dffml.feature import Features
 from dffml.accuracy import Accuracy
 from dffml.model.model import ModelConfig, ModelContext, Model
-from dffml.util.entrypoint import entrypoint
+from dffml.util.entrypoint import entry_point
```
Comment (pdxjohnny) [inline dffml/skel/model/REPLACE_IMPORT_PACKAGE_NAME/misc.py]:
```suggestion
from dffml.util.entrypoint import entrypoint
```

**Comment by pdxjohnny** [comment]
Looking good! Thanks! Just the entrypoint thing.

**Comment by pdxjohnny** [comment]
Oh and it looks like you'll need to run [`black .`](https://github.com/PyCQA/black) from the root of the repo.

File: dffml/skel/source/REPLACE_IMPORT_PACKAGE_NAME/misc.py (hunk 1 above)
Comment (jstar1) [reply dffml/skel/source/REPLACE_IMPORT_PACKAGE_NAME/misc.py]: Got it. Thanks @pdxjohnny.

**Comment by pdxjohnny** [comment]
Thanks @jstar1!

**Comment by jstar1** [comment]
Awesome, anytime @pdxjohnny.

**Pull request merged**